source/features/submission-via-ctrl-enter-everywere.tsx

```diff
@@ -0,0 +1,20 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import {isMac} from '../github-helpers';
+import features from '.';
+
+function addQuickSubmit(): void {
+	select('input#commit-summary-input')!.classList.add('js-quick-submit');
+}
+
+void features.add(import.meta.url, {
+	shortcuts: {
+		[isMac ? 'cmd enter' : 'ctrl enter']: 'Publish a new/edited file',
```

Comment:
I tried to test it by changing the user agent in Chrome but unfortunately it doesn't change accordingly (it should change to <kbd>cmd</kbd><kbd>enter</kbd>) :
![2022-04-21_192555](https://user-images.githubusercontent.com/723651/164509623-8f87a5b3-d7ab-449b-9532-65f7698281f5.jpg)
![2022-04-21_193505](https://user-images.githubusercontent.com/723651/164509677-f1732a21-abdd-462c-b8b3-e6d67a9f371a.jpg)